Cargo.toml

```diff
@@ -14,8 +14,8 @@ cosmwasm-std = { version = "1.1" }
 cw-multi-test = { package = "abstract-cw-multi-test", version = "0.20.2", features = [
   "cosmwasm_1_4",
 ] }
-cw20 = { git = "https://github.com/AbstractSDK/cw-plus.git" }
-cw20-base = { git = "https://github.com/AbstractSDK/cw-plus.git" }
+cw20 = { package = "abstract-cw20", git = "https://github.com/AbstractSDK/cw-plus.git" }
```

Comment:
can't we switch to the actual release of the package instead of the github dep?

Reply:
Sure, thanks